content/en/continuous_integration/intelligent_test_runner/python.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Intelligent Test Runner for Python
+kind: documentation
+further_reading:
+    - link: "/continuous_integration/tests"
+      tag: "Documentation"
+      text: "Explore Test Results and Performance"
+    - link: "/continuous_integration/troubleshooting/"
+      tag: "Documentation"
+      text: "Troubleshooting CI"
+---
+
+{{< callout url="#" btn_hidden="true" >}}Intelligent Test Runner for Python (using `pytest`) in beta.{{< /callout >}}
```

Comment:
Did you intend to put a sign up form or some other URL in this callout?

Reply:
That is a great question. I borrowed it from our Java page ( https://docs.datadoghq.com/continuous_integration/intelligent_test_runner/java/?tab=gradle ), and I don't actually know if we have a form for it, but I'll find out.

Reply:
@brett0000FF , I removed the header (so there's no more callout), and also removed the backticks around `pytest` that weren't rendering. Feel free to resolve this conversation if you're happy with the change. :)

Reply:
@romainkomorndatadog , great--looks good! Let me know if you need me to merge this for you. Oh, for future reference, you should be able to wrap text like pytest in <code></code> to get the monospace formatting within callouts.